beacon-chain/rpc/lookup/blocker_test.go

```diff
@@ -158,172 +161,335 @@ func TestGetBlock(t *testing.T) {
 }
 
 func TestGetBlob(t *testing.T) {
-	params.SetupTestConfigCleanup(t)
-	cfg := params.BeaconConfig().Copy()
-	cfg.DenebForkEpoch = 1
-	params.OverrideBeaconConfig(cfg)
+	const (
+		slot          = 123
+		blobCount     = 4
+		denebForEpoch = 1
+		fuluForkEpoch = 2
+	)
+
+	setupDeneb := func(t *testing.T) {
```

Comment:
Isn't it OK to run the `setupFulu` code once at the start? It shouldn't matter for Deneb tests if the Fulu fork epoch is set

Reply:
It's not OK actually (I tried that first)....
For example, because the "head" test expects the head to be a Deneb block, then if Fulu is setup for this test, the function will expect data column sidecars to be stored, not blob sidecars.

Of course I can modify the "head" test to be Fulu compatible.
But if I do so, I'll probably have the same issue when switching from peerDAS to fullDAS.

Or I can modify the structure of the test not to use
```go
			GenesisTimeFetcher: &testutil.MockGenesisTimeFetcher{
				Genesis: time.Now(),
			},
```

I would prefer not to actually, but if mandatory of course I can change.